js/script.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+var url = 'http://api.icndb.com/jokes/random';
+var button = document.getElementById('get-joke');
+var paragraph = document.getElementById('joke');
+
+function getJoke() {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', url);
+  xhr.addEventListener('load', function(){
+    var response = JSON.parse(xhr.response);
+    paragraph.innerHTML = response.value.joke;
```

Comment:
what will happen when `value.joke` don't exist in server response? use or operator to assign some default value in that case